### ISSUE TITLE
Multiple packets in HRR

### ISSUE THREAD

**Issue by ekr** [quicwg/base-drafts#2169]
```
A client is not required to fit the ClientHello that it sends in response to a
HelloRetryRequest message into a single UDP datagram.
```

I think the reasoning here is that the server creates state at the time it sends the HRR so it's fine to reassemble? If so, we should say that.

**Comment by martinthomson** [comment]
PRs welcome.

**Comment by martinthomson** [comment]
Fixed by #2928.
